web/src/message_edit.js

```diff
@@ -697,7 +697,7 @@ function handle_resolve_topic_failure_due_to_time_limit(topic_is_resolved) {
     });
 }
 
-export function toggle_resolve_topic(message_id, old_topic_name) {
+export function toggle_resolve_topic(message_id, old_topic_name, show_error_in_message_view) {
```

Comment:
I think `in_message_view` doesn't feel like the right way to express the situation, since the place we display these errors is at the top of the screen; maybe something like `report_errors_in_global_banner` might make more sense?